drupal/recipes/wunder_landing_pages/recipe.yml

```diff
@@ -21,10 +21,10 @@ config:
       ensure_exists:
         label: 'Wunder content editor'
       grantPermissions:
-        - 'create wunder_landing_page content'
-        - 'delete any wunder_landing_page content'
-        - 'delete own wunder_landing_page content'
-        - 'edit any wunder_landing_page content'
-        - 'edit own wunder_landing_page content'
-        - 'revert wunder_landing_page revisions'
-        - 'view wunder_landing_page revisions'
+        - 'create landing_page content'
```

Comment:
this is the correct name of the permission. Drupal 10 is more strict and it correctly complained about incorrect permission name